### PR TITLE
feat: add vmbackup known issues to v1.2

### PR DESCRIPTION
**Pull request by FrankYang0529**
ref: https://github.com/harvester/harvester/issues/4612#top

### CODE REVIEW

File: versioned_docs/version-v1.2/vm/backup-restore.md [hunk 1]
```diff
@@ -171,3 +171,13 @@ You can only choose to retain the previous volumes.
...
+
+## Known issues
+
+### VM Backup metadata file name conflicts
+
+When users create a VM Backup, harvester creates a metadata file in the [backup target](../advanced/settings.md#backup-target). The metadata file keeps VM Backup YAML data. The metadata file path is `<storage-path>/harvester/vmbackups/<vmbackup-namespace>-<vmbackup-name>.cfg`.
+
+If users create a VM Backup `c` in `a-b` namespace and another `b-c` in `a` namespace. The metadata file name conflicts, because both are `a-b-c.cfg`. The issue will be fixed in v1.3.0. The metadata file path will be `<storage-path>/harvester/vmbackups/<vmbackup-namespace>/<vmbackup-name>.cfg`.
+
+In a havester version before v1.3.0, users should avoid creating VM Backups which may make the metadata file name conflict.
```
Comment (jillian-maroket) [inline versioned_docs/version-v1.2/vm/backup-restore.md:183]:
```suggestion
### VM Backup Metadata File Naming Conflicts

Whenever you create a VM backup, Harvester generates a metadata file in the [backup target](../advanced/settings.md#backup-target). The metadata file, which is found in `<storage-path>/harvester/vmbackups/<vmbackup-namespace>-<vmbackup-name>.cfg`, contains VM backup data in YAML format.

The naming convention for these metadata files can introduce conflicts. Specifically, files generated for VM backups that were created in different namespaces can have the exact same file name.

Example:
| VM backup name | Namespace | Metadata file name |
| --- | --- | --- |
| `c` | `a-b` | `a-b-c.cfg` |
| `b-c` | `a` | `a-b-c.cfg` |

Harvester v1.3.0 fixes this issue by changing the metadata file path to `<storage-path>/harvester/vmbackups/<vmbackup-namespace>/<vmbackup-name>.cfg`. If you are using an earlier version, however, ensure that VM backup names do not cause the described file naming conflicts.
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

File: versioned_docs/version-v1.2/vm/backup-restore.md (hunk 1 above)
Comment (FrankYang0529) [reply versioned_docs/version-v1.2/vm/backup-restore.md]: Hi @jillian-maroket, thanks for the review. I updated the document, but changing one line. The data is stored in JSON format.

**Review by jillian-maroket — COMMENTED** [review]
LGTM. Thanks!

**Review by bk201 — APPROVED** [review]

**Comment by FrankYang0529** [comment]
> @FrankYang0529 Please co-author Jillian in the commit message.

Update commit message. Thanks.

**Pull request merged**